.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+version: 2
+
+defaults: &defaults
+    working_directory: ~/repo
+
+jobs:
+  build:
+    environment:
+    # Provide Code Climate Test Reporter ID
+      CC_TEST_REPORTER_ID: 91655fa6fb15027379ee6f678a0387485497cbb58e2b8a4ed6701c050fc04549
```

Comment:
Great work. However, it is best we set the Test Reporter ID as an environment variable at Circle CI settings. This won't allow other people to connect to our repo through it. It would be good not to commit the repo's secrets.